include/spdk_internal/bdev.h

```diff
@@ -213,6 +213,12 @@ struct spdk_bdev_io {
 		struct {
 			enum spdk_bdev_reset_type type;
 		} reset;
+		struct {
+			void *cmd;
```

Comment:
I agree with Daniel - don't store a pointer and store a struct spdk_nvme_cmd directly. Make a copy from the user's command.